content/english/useless-box/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Useless box: Silly gadget circuit"
+description: "A guide to build a 'useless box' to learn the basics of circuits and how a battery powers action."
+date: 2020-09-207T11:45:38-07:00
```

Comment:
@veronicagg typo in the dates - they should be:

```suggestion
date: 2022-09-24T11:45:38-07:00
```

Then for some reason the workshop end card shows up correctly